test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -192,6 +261,1427 @@ public void testCreateDeleteKeyValueTable() {
 
     }
 
+    /**
+     * Test case-42: Insert Same Key into multiple KeyFamilies
+     */
+    @Test
+    public void testSameKeyMutipleKF() {
```

Comment:
This test is already covered in the base class.

Reply:
Okay, I'll remove this test as its a duplicate.